### DISCUSSION TITLE
Drop docker notebooks image build

### DISCUSSION THREAD

**Discussion by IceKhan13** [Qiskit/qiskit-serverless#1046] [Polls]
Currently we build and push docker images for example notebooks. They are extremely heavy and it looks like usage it very low (if any at all).

**Comment by psschwei** [comment]
One place we are using it is in the notebook tests: https://github.com/Qiskit-Extensions/quantum-serverless/blob/22a4e8b738f8cfa5ffc5282a497ffc18ab76a0d0/.github/workflows/docker-build.yaml#L22 , but that's not using the actual image (rather building from scratch), so I don't think that would be an issue.

Longer term, it shouldn't be too bad to refactor that test and run the notebooks locally rather than in a container (I think that's what I did when trying to test on kubernetes), but I don't think that's super high on the priority list right now...

**Reply by Tansito** [reply]
I agree, I would continue maintaining it for development purposes but the build step is something that we can afford us.
